english/little-me/Apple/Apple.md

```diff
@@ -0,0 +1,10 @@
+Thank you so much.
+Now that I am full I can rewind the time.
+
+Do you like to see my powers?
+
+You Choosed To Say:
+
+[Yes](https://github.com/udacity/create-your-own-adventure/tree/master/english/little-me/little-me.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.